programs/keeper-client/KeeperClient.cpp

```diff
@@ -313,6 +330,26 @@ void KeeperClient::runInteractive()
     }
 }
 
+void KeeperClient::runInteractiveInputStream()
+{
+    for (String input; std::getline(std::cin, input);)
+    {
+        if (!processQueryText(input))
+            break;
+
+        std::cout << "\a\a\a\a" << std::endl;
```

Comment:
This is a hack that hopefully will be removed after supporting JSON output format.